runtime/ftplugin/obl.vim

```diff
@@ -0,0 +1,58 @@
+" Vim filetype plugin file
+" Language:    Oblivion Language (obl)
+" Original Creator: Kat <katisntgood@gmail.com>
+" Maintainer:  Kat <katisntgood@gmail.com>
+" Created:     August 08, 2021
+" Last Change: 13 November 2022
+
```

Comment:
Could you please set `b:undo_ftplugin` to undo the configuration changes?

See `:help undo_ftplugin` for details and https://github.com/vim/vim/tree/master/runtime/ftplugin for examples?